### DISCUSSION TITLE
Android端如何让一个pagview里面的文本框里面的内容居中

### DISCUSSION THREAD

**Discussion by zhongbingrong** [Tencent/libpag#175] [Q&A] [answered]
![middle_img_v2_b4fdcfcf-d9e0-4ed3-8b48-8cc50026e66g](https://user-images.githubusercontent.com/49142348/158720040-0dafc08a-eefb-41ec-aa13-ea5ff623d2e4.png)

**Comment by zhongbingrong** [comment]
锁屏充电动画用pag实现的，如上面的图，是设计同事提供的一个pag动画，如上面当电量是98的时候做替换的时候，用如下方法不能居中，只会从“1”的位置开始显示
textData.justification = PAGText.Justification.FullJustifyLastLineCenter;

想的是文本框里面的内容居中，文本框本身是居中的

**Comment by zhongbingrong** [comment, marked as answer]
@kevingpqi123 可以帮忙解释一下吗

**Reply by zhongbingrong** [reply]
已经解决

**Reply by BerylLuo** [reply]
哈喽，想请问一下是怎么解决的呀？

**Reply by kevingpqi123** [reply]
这个更多的和文件设计有关，可以把导出的 pag 文件发出来看下

**Reply by zhongbingrong** [reply]
> 呀
是的，是PAG文件有关。

**Reply by daiziyuBigo** [reply]
请问这个应该设置什么
